app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -48,3 +48,11 @@ fun setImageDrawable(imageView: ImageView, thumbnailGraphic: LessonThumbnailGrap
     }
   )
 }
+
+@BindingAdapter("profile:src")
+fun setProfileImage(imageView: ImageView, imageUrl: String) {
+  Glide.with(imageView.context)
+    .load(imageUrl)
+    .placeholder(R.drawable.default_avatar)
```

Comment:
Reminder to pull in a vector version of this instead of PNG.

Reply:
Removed PNG and added a vector asset.